### ISSUE TITLE
LF target properties 'docker' + 'tracing' fails to connect to Telegraf

### ISSUE THREAD

**Issue by elgeeko1** [xronos-inc/xronos-dashboard#1]
The combination of target properties `docker` and `tracing` won't work out of the gate with the default configuration of the Xronos Dashboard. The issue is that Xronos Dashboard `docker-compose.yml` uses the docker network `xronos-dashboard` and the LF generated `docker-compose.yml` file is `lf`.

The tracing plugin defaults to `0.0.0.0` for the host, but the firewall prevents a container from accessing ports on the host. If the environment variable `LF_TELEGRAF_HOST_NAME` is set to `telegraf`, then the containers will need to be in the same docker network.

One way to resolve this is to modify the generated docker-compose file to pass in the value of `LF_TELEGRAF_HOST_NAME` from the environment using interpolation, i.e.

```docker
env:
  LF_TELEGRAF_HOST_NAME: ${LF_TELEGRAF_HOST_NAME:-"0.0.0.0"}
```

**Comment by elgeeko1** [comment]
Example use cases the user needs to specify where Telegraf is running:

1. "0.0.0.0" if they are running Xronos Dashboard from docker compose and then running native LF apps. (this works now)
2. "telegraf" if they are running Xronos Dashboard and LF apps in the same docker network.
3. "host.docker.internal" if they are running Xronos Dashboard and LF apps in different docker networks.
4. "host.container.internal" if they are running podman
5. IP/address of an external server if Xronos Dashboard is running on another host (AVP is an example)

(1) works now. The pre-run script in Docker will resolve this issue for (2), (4) and (5) but not (3).

I believe (3) should be the default since the dashboard should run independently of an application, since multiple users / applications / hosts may be using the dashboard at the same time. It should be a standalone service.

**Comment by lhstrh** [comment]
Addressed with changes in Lingua Franca v0.8.0. Specifically:
 - https://github.com/lf-lang/lingua-franca/pull/2349
 - https://github.com/lf-lang/lingua-franca/pull/2234
 - https://github.com/lf-lang/lingua-franca/pull/2345
